core/templates/domain/collection/collection.model.ts

```diff
@@ -32,35 +32,39 @@ interface ExplorationIdToNodeIndexMap {
 }
 
 export interface CollectionBackendDict {
-  'id': string;
-  'title': string;
-  'objective': string;
-  'language_code': string;
-  'tags': string[];
-  'schema_version': number;
+  // Creating an Empty Collection initializes properties below with
```

Comment:
Maybe elaborate this a little more? When/why are empty collections created?

Reply:
done